doc/MANUAL.md

```diff
@@ -179,62 +318,65 @@ This option allows you to specify the filename for the report, rather than havin
+
+1.  `--format csv` - write output file in csv (comma separated) format.
+
+```csv
+vendor,product,version,cve_number,severity,remarks,comments
+haxx,curl,7.34.0,CVE-2014-0015,MEDIUM,Mitigated,
+haxx,curl,7.34.0,CVE-2014-0138,MEDIUM,NewFound,
+haxx,curl,7.34.0,CVE-2014-0139,MEDIUM,Unexplored,
 ```
-2. `--format json` - prints in json (javascript object notation) format.
+
+2.  `--format json` - write output file in json (javascript object notation) format.
+
 ```json
 [
-    {
-        "package": "libgcrypt",
-        "version": "1.6.0",
-        "cve_number": "CVE-2017-9526",
-        "severity": "MEDIUM"
-    },
-    {
-        "package": "libgcrypt",
-        "version": "1.6.0",
-        "cve_number": "CVE-2018-0495",
-        "severity": "MEDIUM"
-    },
-    {
-        "package": "libgcrypt",
-        "version": "1.6.0",
-        "cve_number": "CVE-2018-6829",
-        "severity": "HIGH"
-    }
+  {
+    "vendor": "haxx",
+    "product": "curl",
+    "version": "7.34.0",
+    "cve_number": "CVE-2014-0015",
+    "severity": "MEDIUM",
+    "remarks": "Mitigated",
+    "comments": ""
+  },
+  {
+    "vendor": "haxx",
+    "product": "curl",
+    "version": "7.34.0",
+    "cve_number": "CVE-2014-0138",
+    "severity": "MEDIUM",
+    "remarks": "NewFound",
+    "comments": ""
+  },
 ]
 ```
-3. `--format console` - prints in nice tabular format.
-```console
-+=================================================================+
-|   CVE Binary Tool Report Generated: 2020-04-29  10:04:06        |
-+=================================================================+
-
-+=================================================================+
-|   MODULE NAME      |  VERSION  |    CVE NUMBER      | SEVERITY  |
-+=================================================================+
-| bzip2              | 1.0.2     | CVE-2005-0953      | LOW       |
-+--------------------+-----------+--------------------+-----------+
-| bzip2              | 1.0.2     | CVE-2005-1260      | MEDIUM    |
-+--------------------+-----------+--------------------+-----------+
-| bzip2              | 1.0.2     | CVE-2008-1372      | MEDIUM    |
-+--------------------+-----------+--------------------+-----------+
 
-```
+3.  `--format console` - prints in nice colored tabular format.
+<figure>
+  <img src="https://i.imgur.com/UwH6vA7.png"
+    alt="
+    cve-bin-tool: Report Generated: 2020-07-31  17:49:56
+    1. NewFound CVEs:
+    Vendor, Product, Version, CVE Number   , Severity
+    haxx  , curl   , 7.34.0 , CVE-2014-0138, HIGH      
+    haxx  , curl   , 7.34.0 , CVE-2014-0139, CRITICAL
+    haxx  , curl   , 7.34.0 , CVE-2014-0015, MEDIUM
+    " 
+    style="width:100%;white-space:pre;">
+  <figcaption>formated console output</figcaption>
+</figure>
```

Comment:
I don't have any easy way to do this in markdown. So, I am using html here since github supports it.